MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/MediaListView.js

```diff
@@ -77,15 +124,21 @@ class MediaListView extends mix(AbstractDataTableView).with(UrlPaginateViewMixin
     /**
      * @inheritDoc
      */
-    getColumnsDefinition() {
-        return [];
+    generateUrlUpdatePage(page) {
+       return Backbone.history.generateUrl('listMedia', {page : page});
```

Comment:
I'm not sure to understand. If I got sorting on page 1, clicking on page 2, keep the sort.